### DISCUSSION TITLE
nuxt pre-render page not that i want

### DISCUSSION THREAD

**Discussion by Tonight11** [nuxt/nuxt#19714] [Questions]
So i want to pre-render only my landing page, but nuxt pre-prender every other pages, so i lose reactivity. What should i do?

`nitro: {
		prerender: {
			routes: ['/landing'],
		},
	},`

![Screenshot_1](https://user-images.githubusercontent.com/64700669/225390901-fde45719-1812-437d-8ecb-543abcd7da62.png)

**Comment by danielroe** [comment]
You can set `nitro.prerender.ignore` in your `nuxt.config`.

**Reply by Tonight11** [reply]
could ignore every others page except landing page?

**Comment by zernonia** [comment]
```
    prerender: {
      crawlLinks: false,
      routes: ["/landing"],
    },
```

You can ignore crawling other page by setting `nitro.prerender.crawlLinks` to `false`.
